core/src/lib/util.h

```diff
@@ -75,5 +77,33 @@ void StringToLowerCase(std::string& s);
 void StringToLowerCase(std::string& out, const std::string& in);
 bool pm_append(void* pm_string, const char* fmt, ...);
 
+class BoolString {
+ public:
+  BoolString(const std::string& s) : str_value(s)
+  {
+    if (true_values.find(str_value) == true_values.end()
+        && false_values.find(str_value) == false_values.end()) {
+      std::string err = {"Wrong parameter: "};
+      throw std::out_of_range(err + str_value);
```

Comment:
As we tried to stick to the google style guide, I would like to refer to:
https://google.github.io/styleguide/cppguide.html#Exceptions